src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -729,26 +784,29 @@ static Library()
             {
                 BuiltinFunctionsCore.Ln,
                 StandardErrorHandling<NumberValue>(
+                    BuiltinFunctionsCore.Ln.Name,
                     expandArguments: NoArgExpansion,
                     replaceBlankValues: ReplaceBlankWithZero,
                     checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
-                    checkRuntimeValues: StrictNumericPositiveNumberChecker,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: Ln)
             },
             {
                 BuiltinFunctionsCore.Log,
                 StandardErrorHandling<NumberValue>(
+                    BuiltinFunctionsCore.Log.Name,
                     expandArguments: InsertDefaultValues(outputArgsCount: 2, fillWith: new NumberValue(IRContext.NotInSource(FormulaType.Number), 10)),
                     replaceBlankValues: ReplaceBlankWithZero,
                     checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
-                    checkRuntimeValues: StrictNumericPositiveNumberChecker,
```

Comment:
Why is this being removed?

Reply:
There is no need for an additional check - the function will return a non-finite number, and the wrapper will create the appropriate error